README.md

```diff
@@ -36,18 +33,17 @@ ignored by the version control system.
 
 ##### Running a Single Test
 
-If you haven't already done so, you will need to configure the HCA CLI tool to use the EBI installation of the upload service rather than the DCP (default) version.
+To run a single test, make sure that all necessary environment variables are provided.
```

Comment:
I tried to setup using the instructions here and got an error about missing 'client_email'.

Reply:
Could we take this offline, please?
As it is perfectly working for all of us, but you.
Tx